backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -3725,4 +3725,51 @@ databaseChangeLog:
             GRANT SELECT ON ${database.defaultSchemaName}.result TO ${noPhiUsername};
       rollback:
         sql: |
-          ALTER TABLE ${database.defaultSchemaName}.result DROP COLUMN test_result;
\ No newline at end of file
+          ALTER TABLE ${database.defaultSchemaName}.result DROP COLUMN test_result;
+  - changeSet:
+      id: add-test-correction-columns
+      author: nathan@skylight.digital
+      comment: Adds columns in support of test correction workflow
+      changes:
+        - tagDatabase:
+            tag: add-test-correction-columns
+        - addColumn:
+            tableName: test_order
+            columns:
+              - column:
+                  name: corrected_by
+                  type: uuid
+                  constraints:
+                    nullable: true
+                    foreignKeyName: fk__test_order__api_user
+                    references: api_user(internal_id)
+              - column:
+                  name: corrected_at
+                  type: DATETIME
+                  constraints:
+                    nullable: true
+              - column:
+                  name: additional_correction_information
```

Comment:
In case you missed it in the PR notes, please consider:

<img width="679" alt="Screen Shot 2022-04-12 at 4 15 56 PM" src="https://user-images.githubusercontent.com/27730981/163046253-e5f36bde-a21a-422c-a628-9c221445f2ef.png">

Reply:
See https://github.com/CDCgov/prime-simplereport/pull/3650#discussion_r849864098 for more discussion.